data_handler/handlers/order_books/myswap/main.py

```diff
@@ -52,6 +50,8 @@ async def _async_fetch_price_and_liquidity(self) -> None:
         """Fetch price and liquidity data from the MySwap CLMM service."""
         all_pools = self.connector.get_pools_data()
         filtered_pools = self._filter_pools_data(all_pools)
+        if not filtered_pools:
+            self.logger.info(f"No pools for pair: {self.token_a} - {self.token_b}")
```

Comment:
This isn't expected, right? So maybe use the `warning` level .. what do you think?